wiki/PlayersManual.md

```diff
@@ -350,25 +350,35 @@ Trading panel: | &nbsp;
 :--- | :---
 Buy | +, Return, or Space
 Sell | -, Delete, or Backspace
-Buy / sell 5x | Shift
-Buy / sell 20x | Control / Command
-Buy / sell 500x | Option / Alt
 Buy / sell all | Shift + B / S
 View commodity price map | Map key
 
-The multiplier modifiers (Shift = 5x, Command / Control = 20x, and Option / Alt = 500x) also work in the hiring panel, the shipyard, and the outfitter. And, they can be combined together - for example, Shift + Control to buy or sell 100 items at a time.
+Multipliers: | &nbsp;
+:--- | :---
+5x | Shift
+20x | Control / Command
+500x | Option / Alt
+
+The multiplier modifiers work in the Trading panel, Hiring panel, Shipyard and Outfitter. They can be used in combination - for example, Shift + Control to buy or sell 100 items at a time. All three used together will provide a whopping 50,000x multiplier.
```

Comment:
"multiplier modifier" is a little bit of a mouthful. Also, an em-dash is best here because GitHub (unlike ES) actually has unicode support.
```suggestion
These multipliers work in the Trading panel, Hiring panel, Shipyard and Outfitter. They can be used in combination—for example, Shift + Control to buy or sell 100 items at a time. All three used together will provide a whopping 50,000x multiplier.
```